lib/tasks/cucumber.rake

```diff
@@ -41,10 +53,12 @@ begin
       ::CodeStatistics::TEST_TYPES << "Cucumber features" if File.exist?('features')
     end
   end
-  desc 'Alias for cucumber:ok'
-  task :cucumber => 'cucumber:ok'
 
+  desc 'Alias for cucumber:ok'
   task :default => :cucumber
+  task :cucumber => 'cucumber:aa'
```

Comment:
you mean by running `cucumber` you want it to run the `hq` first and then the `aa` profiles?
That is only possible if we set a rake task for it.  like `$rake cucumber` that will invoke the 2 cucumber commands one after another. like: `$ cucumber -p hq && cucumber -p aa`

(The commands available at the moment are at the top of the page)